### ISSUE TITLE
How to send quoted reply

### ISSUE THREAD

**Issue by SafwanGanz** [telegraf/telegraf#1736]

**Comment by MKRhere** [comment]
Glad you asked! There are several ways. You can do something like this:

```TS
ctx.sendMessage(text, { reply_to_message_id: ctx.message.message_id });
```

But the recommended way is to use `ctx.reply`. You may realise that `ctx.reply` doesn't actually reply. This is something that will change in telegraf v5. Until v5, you can enable this behaviour across the bot by doing this:

```TS
import { useNewReplies } from "telegraf/future";

bot.use(useNewReplies()); // enable this once

bot.on("text", ctx => {
		ctx.reply("Hello!"); // this actually sends a quoted reply now!
		ctx.sendMessage("Hello!"); // old behaviour, sends a message in current chat
});
```

This feature landed in Telegraf [4.9.0 (release notes)](https://github.com/telegraf/telegraf/releases/tag/v4.9.0)

**Comment by SafwanGanz** [comment]
> 

Thanks
